tock/hours/migrations/0018_auto_20160708_0207.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hours', '0017_auto_20160708_0156'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='reportingperiod',
+            old_name='working_hours',
```

Comment:
Back in the day it used to be things like this required an extra data migration, so you'd have the initial schema migration to create the new field, a data migration to run immediately after to shift the data from the old place to the new, and then another schema migration to remove the old field.  Looks like this is improved/simplified now, which is awesome.